README.md

```diff
@@ -1,3 +1,31 @@
+# VSCode WSL Remote Xdebug Link Support
+
+This package now supports generating Xdebug file links for VS Code running in WSL (Windows Subsystem for Linux) via the `vscode-wsl` editor type.
+
+## Usage
+
+Set the editor link template to `vscode-wsl`:
+
+```php
+$debugbar->setEditorLinkTemplate('vscode-wsl');
```

Comment:
Debugbar doesn't have this method(it's in collectors), this documentation is wrong,
although it would be a good idea to do it, I will try to open a PR

`%w` seems to be fixed for the environment, you could do:
```php
$collector->setXdebugLinkTemplate('vscode://vscode-remote/wsl+Ubuntu/%f:%l');
```